src/org/etools/j1939_84/controllers/part03/Part03Step16Controller.java

```diff
@@ -54,12 +58,41 @@
     @Override
     protected void run() throws Throwable {
         // 6.3.16.1.a. Turn the engine off.
+        incrementProgress("Part 3, Step 16 Turn Engine Off and keep the ignition key in the off position");
+        ensureKeyOffEngineOff();
+
         // 6.3.16.1.b. Confirm Fault A is still implanted according to the manufacturer’s instruction.
+        incrementProgress("Waiting for implant of Fault A according to the engine manufacturer's instruction");
```

Comment:
I noticed the first `incrementProgress()` has the Part and Step number before the message.  I like that.  I think we should do that in all cases where possible.